src/Components/table.js

```diff
@@ -8,68 +7,52 @@ import TableHead from '@material-ui/core/TableHead';
+`;
+const TableBodyCellStyled = styled(TableCell)`
+        font-size: 14;
+`;
 
-const StyledTableRow = withStyles((theme) => ({
-    root: {
-        '&:nth-of-type(odd)': {
-            backgroundColor: theme.palette.action.hoverOpacity,
-        },
-    }
-}))(TableRow);
+const TableStyled = styled(Table)`
+    min-width: 100px;
+    margin-top: 10px;
+`;
 
-const useStyles = makeStyles({
-    table: {
-        minWidth: 100,
-    },
-    footer: {
-        background: '#CED2AA',
-        display: "-ms-inline-flexbox",
-    }
-});
-
-export default function Tables() {
-
-    const classes = useStyles();
+export default function ListOfUsers() {
 
     const [rows] = useState([])
 
     return (
         <div>
             <TableContainer component={Paper}>
-                <Table className={classes.table} aria-label="customized table">
+                <TableStyled aria-label="customized table">
                     <TableHead>
                         <TableRow>
-                            <StyledTableCell>Name</StyledTableCell>
-                            <StyledTableCell align="right">Email</StyledTableCell>
-                            <StyledTableCell align="right">Date Created</StyledTableCell>
-                            <StyledTableCell align="right">Date Modified</StyledTableCell>
+                            <TableHeadCellStyled>Name</TableHeadCellStyled>
+                            <TableHeadCellStyled align="right">Email</TableHeadCellStyled>
+                            <TableHeadCellStyled align="right">Date Created</TableHeadCellStyled>
+                            <TableHeadCellStyled align="right">Date Modified</TableHeadCellStyled>
                         </TableRow>
                     </TableHead>
                     {rows.map((row, index) => (
                         <TableBody>
-                            <StyledTableRow key={index}>
-                                <StyledTableCell component="th" scope="row"><Checkbox
+                            <TableRow key={index}>
+                                <TableBodyCellStyled component="th" scope="row"><Checkbox
                                     color={"default"}/> {row.firstName + '  ' + row.lastName}
-                                </StyledTableCell>
-                                <StyledTableCell align="right">{row.Email}</StyledTableCell>
-                                <StyledTableCell align="right">{row.DateCreated}</StyledTableCell>
-                                <StyledTableCell align="right">{row.DateModified}</StyledTableCell>
-                            </StyledTableRow>
+                                </TableBodyCellStyled>
+                                <TableBodyCellStyled align="right">{row.Email}</TableBodyCellStyled>
+                                <TableBodyCellStyled align="right">{row.DateCreated}</TableBodyCellStyled>
+                                <TableBodyCellStyled align="right">{row.DateModified}</TableBodyCellStyled>
```

Comment:
Javascript is camel case, so decapitalize `row.Email`, `row.DateCreated` and `DateModified`.
Also`users` in better indicating purpose of the state variable that `rows`, use it instead.